cmd/kubeadm/app/phases/kubelet/flags.go

```diff
@@ -50,7 +50,7 @@ type kubeletFlagsOpts struct {
 // (from lower to higher):
 // - actual hostname
 // - NodeRegistrationOptions.Name (same as "--node-name" passed to "kubeadm init/join")
-// - "hostname-overide" flag in NodeRegistrationOptions.KubeletExtraArgs
```

Comment:
looks good to me

another File Has the Similar Problem
https://github.com/kubernetes/kubernetes/pull/106998